rasa_sdk/executor.py

```diff
@@ -175,6 +163,8 @@ def _import_submodules(self, package, recursive=True):
                 self._import_submodules(full_name)
 
     def register_package(self, package):
```

Comment:
types here, I guess we can add them in future though

Reply:
added a couple more :+1: